packages/components/src/components/navigation-item/navigation-item.ts

```diff
@@ -0,0 +1,297 @@
+
+  /** *Vertical Only: Adds additional padding to navigation item's left and right sides. */
+  @property({ type: Boolean, reflect: true }) divider = false;
+
+  /** *Vertical Only if 'children' slot and no 'href': Reflects HTML details element state and allows control from parent. */
+  @property({ type: Boolean, reflect: true }) open = false;
+
+  private handleMouseEnter() {
+    this.emit('sd-mouse-enter');
+  }
+
+  private handleMouseLeave() {
+    this.emit('sd-mouse-leave');
+  }
+
+  private handleClickSummary(event: MouseEvent) {
+    event.preventDefault();
+    event.stopPropagation();
+
+    if (!this.disabled) {
+      this.open = !this.open;
+      this.emit('sd-click', { detail: { open: this.open } });
+    }
+  }
+
+  private handleClickButton(event: MouseEvent) {
+    if (this.disabled) {
+      event.preventDefault();
+      event.stopPropagation();
+    } else {
+      this.emit('sd-click');
+    }
+  }
+
+  private isButton(): boolean {
+    return !this.href && !this.hasSlotController.test('children');
+  }
+
+  private isLink(): boolean {
+    return !!this.href;
+  }
+
+  private isAccordion(): boolean {
+    return !this.href && this.hasSlotController.test('children');
+  }
+
+  private calculatePaddingX(): string {
+    if (this.horizontal) return '';
+    if (this.relaxed && this.indented) return 'pl-8 pr-4';
+    if (this.relaxed) return 'px-4';
+    if (this.indented) return 'pl-4';
+    return '';
+  }
+
+  render() {
+    const slots = {
+      label: this.hasSlotController.test('[default]'),
+      'icon-left': this.hasSlotController.test('icon-left'),
+      'icon-right': this.hasSlotController.test('icon-right'),
+      main: this.hasSlotController.test('main'),
+      description: this.hasSlotController.test('description'),
+      children: this.hasSlotController.test('children')
+    };
+    const tag = this.isLink() ? literal`a` : slots['children'] ? literal`summary` : literal`button`;
+
+    // styles
+    const horizontalPaddingBottom = this.horizontal ? 'pb-2' : 'pb-3';
+    const hostDisplayStyle = this.horizontal
+      ? html`<style>
+          :host {
+            display: inline-block;
+          }
+        </style>`
+      : html`<style>
+          :host {
+            display: block;
+          }
+        </style>`;
+
+    // conditional elements
+    const chevron =
+      (this.chevron || slots['children']) && !this.horizontal
+        ? html`<sd-icon
+            name="chevron-down"
+            part="chevron"
+            library="system"
+            color="currentColor"
+            class=${cx('h-6 w-6', this.isAccordion() ? (this.open ? 'rotate-180' : 'rotate-0') : 'rotate-[270deg]')}
+          ></sd-icon>`
+        : null;
+
+    const divider =
+      this.divider && !this.horizontal
+        ? html`<sd-divider
+            part="divider"
+            class=${cx(
+              'w-[calc(100%-64px)] absolute top-[-1px] left-0',
```

Comment:
Why do you need the width calculation? Wouldn't it work if use the padding you already calculated etc.?

Reply:
This is a detail I noticed in the design.  The divider sits above the element, not flush with the top.  In order to achieve that using the sd-divider, it sits outside of the "tag" element (button / anchor / summary) and additionally outside of the details wrapper (for accordion variant) with absolute position.

This means I must add an additional set of cases to calculatePaddingX OR compensate by adjusting the width.  I went with this one line solution as opposed to the 6 line solution, but I'll adjust it.